js/setup.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var NAMES = ['Иван', 'Хуан Себастьян', 'Мария', 'Кристоф', 'Виктор', 'Юлия', 'Люпита', 'Вашингтон'];
+var SURNAMES = ['да Марья', 'Верон', 'Мирабелла', 'Вальц', 'Онопко', 'Топольницкая', 'Нионго', 'Ирвинг'];
+var COAT_COLORS = ['rgb(101, 137, 164)', 'rgb(241, 43, 107)', 'rgb(146, 100, 161)', 'rgb(56, 159, 117)', 'rgb(215, 210, 55)', 'rgb(0, 0, 0)'];
+var EYE_COLORS = ['black', 'red', 'blue', 'yellow', 'green'];
+var WIZZARDS_COUNT = 4;
+
+var setup = document.querySelector('.setup');
+var setupSimilar = document.querySelector('.setup-similar');
```

Comment:
можно ограничить поиск и искать внутри setup
```suggestion
var setupSimilar = setup.querySelector('.setup-similar');
```

Reply:
Готово